homework03/src/main/kotlin/ru/lpfun/spring/homework03/parser/QuestionsCsvParserImpl.kt

```diff
@@ -0,0 +1,25 @@
+package ru.lpfun.spring.homework03.parser
+
+import org.springframework.core.io.ClassPathResource
+import org.springframework.stereotype.Service
+import ru.lpfun.spring.homework03.common.interfaces.Parser
+import java.io.InputStreamReader
+import java.util.*
+
+@Service
+class QuestionsCsvParserImpl : Parser<QuestionCsv> {
+    override fun parse(path: String): List<QuestionCsv> {
+        val csvFileExtension = ".csv"
+        val localizedFilePath = "${path.removeSuffix(csvFileExtension)}_${Locale.getDefault()}$csvFileExtension"
```

Comment:
Для дефолтной английской локали:
```
Exception in thread "main" java.io.FileNotFoundException: class path resource [data/questions_en_US.csv] cannot be opened because it does not exist
```

Reply:
язык задается в настройках